.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I feel this particular file should be:
* removed
* `.cache` should be added to gitignore if any of our tools generate contents in such path